### PR TITLE
Training and Loading from Menu

### PR DESCRIPTION
**Pull request by alfredofernandez2021**
Branch will introduce training and loading features for MNIST dataset, accessible from main menu

### CODE REVIEW

**Comment by alfredofernandez2021** [comment]
Welcome to the Main Menu!
1) Create Neural Network
2) Load Neural Network
3) Introduction and Info
4) Exit Network Manager
Selection: 2

Loading:
Enter XML file name to load from: test1.xml

Manage:
1) Select DataSets
2) Run Training
3) Run Testing
4) Save Solution
5) Help
6) Back
Selection: 1

Dataset:
Training set image file path: a
Training set label file path: a
Testing set image file path: a
Testing set label file path: a

Manage:
1) Select DataSets
2) Run Training
3) Run Testing
4) Save Solution
5) Help
6) Back
Selection: 6

Welcome to the Main Menu!
1) Create Neural Network
2) Load Neural Network
3) Introduction and Info
4) Exit Network Manager
Selection: 4

<Exits from main with code 0>

**Comment by alfredofernandez2021** [comment]
Welcome to the Main Menu!
1) Create Neural Network
2) Load Neural Network
3) Introduction and Info
4) Exit Network Manager
Selection: 2

Loading:
Enter XML file name to load from: test5.xml

Manage:
1) Select DataSets
2) Run Training
3) Run Testing
4) Save Solution
5) Help
6) Back
Selection: 1

Dataset:
Training set image file path: a
Training set label file path: a
Testing set image file path: a
Testing set label file path: a

Manage:
1) Select DataSets
2) Run Training
3) Run Testing
4) Save Solution
5) Help
6) Back
Selection: 2

Training:
Batch size: 1
Learning rate: 1
Current score: 0.15
answer: 0       correct: 5

Current score: 0.115
answer: 0       correct: 1

Current score: 0.12
answer: 0       correct: 8
...

Current score: 0.099665
answer: 5       correct: 4

Current score: 0.0995987
answer: 5       correct: 5

Current score: 0.0995826
answer: 5       correct: 4

Final score: 0.09955
Type 0 to exit:
0

Manage:
1) Select DataSets
2) Run Training
3) Run Testing
4) Save Solution
5) Help
6) Back
Selection: 6

Welcome to the Main Menu!
1) Create Neural Network
2) Load Neural Network
3) Introduction and Info
4) Exit Network Manager
Selection: 4

<Program terminates with main returning 0>
